app/assets/javascripts/quick_create.js

```diff
@@ -0,0 +1,93 @@
+function addToAutocomplete(json) {
+  var source = JSON.parse($("#" + json.autocomplete_id).attr("data-autocomplete-source"));
+  source.unshift(json.name)
```

Comment:
It could, but then we would have make an Ajax call, and it seems unnecessary to me if it's already on the DOM